src-tauri/src/northstar/install.rs

```diff
@@ -96,6 +96,22 @@ async fn do_install(
     log::info!("Extracting Northstar...");
     extract(nfile, std::path::Path::new(&extract_directory))?;
 
+    if game_install.profile != "R2Northstar" {
+        // We are using a non standard Profile, we must:
+        // - move the DLL
+        // - rename the Profile
+
+        // Move DLL into default Profile
+        let old_dll_path = format!("{}/Northstar.dll", extract_directory);
```

Comment:
Maybe same with `Northstar.dll`. Or at the very least define it as a variable at the beginning of the codeblock and then just re-use the variable.